src/extensions/npm/extensions/package.json

```diff
@@ -16,8 +16,8 @@
     "name": "OpenLens Authors"
   },
   "dependencies": {
-    "@material-ui/core": "4.12.3",
-    "@types/node": "14.17.14",
+    "@material-ui/core": "^4.12.3",
+    "@types/node": "^16.11.58",
```

Comment:
These are changed to make them the same as what we have in the root package.json